controller/api/serializers.py

```diff
@@ -22,46 +23,64 @@
 TAGVAL_MATCH = re.compile(r'^\w+$')
 
 
-class OwnerSlugRelatedField(serializers.SlugRelatedField):
-    """Filter queries by owner as well as slug_field."""
-
-    def from_native(self, data):
-        """Fetch model object from its 'native' representation.
-        TODO: request.user is not going to work in a team environment...
-        """
-        self.queryset = self.queryset.filter(owner=self.context['request'].user)
-        return serializers.SlugRelatedField.from_native(self, data)
-
-
-class JSONFieldSerializer(serializers.WritableField):
-    def to_native(self, obj):
+class JSONFieldSerializer(serializers.Field):
+    def to_representation(self, obj):
         return obj
 
-    def from_native(self, value):
+    def to_internal_value(self, data):
         try:
-            val = json.loads(value)
+            val = json.loads(data)
         except TypeError:
-            val = value
+            val = data
         return val
 
 
-class UserSerializer(serializers.ModelSerializer):
-    """Serialize a User model."""
+class ModelSerializer(serializers.ModelSerializer):
 
+    uuid = serializers.ReadOnlyField()
+
+    def get_validators(self):
+        """
+        Hack to remove DRF's UniqueTogetherValidator when it concerns the UUID.
+
+        See https://github.com/deis/deis/pull/2898#discussion_r23105147
```

Comment:
I think this is a better fix, even if it is a hack. Leaves the the schema intact and allows us to change the behavior in one place if DRF's validation behavior changes in this regard.